# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Adam Pribyl <pribyl@lowlevel.cz>, 2008-2009
# Adam Přibyl <pribyl@lowlevel.cz>, 2008
# Adam Slovacek <adamslovacek@gmail.com>, 2013
# Filip Hruska <fhr@fhrnet.eu>, 2013
# Filip Hruska <fhr@fhrnet.eu>, 2015
# FooBar <thewired@riseup.net>, 2015
# Jan Splíchal <splichal.jan@gmail.com>, 2014
# Jiří Vírava <appukonrad@gmail.com>, 2014-2015,2017
# dope <maniak11@gmail.com>, 2013
# Michal Várady <miko.vaji@gmail.com>, 2015
# Mikulas Holy, 2017
# Radek Bensch <inactive+Radog@transifex.com>, 2013
msgid ""
msgstr ""
"Project-Id-Version: The Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-11-10 15:57+0100\n"
"PO-Revision-Date: 2017-11-11 03:09+0000\n"
"Last-Translator: carolyn <carolyn@anhalt.org>\n"
"Language-Team: Czech (http://www.transifex.com/otf/torproject/language/cs/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: cs\n"
"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"

#: ../tails_installer/creator.py:101
msgid "You must run this application as root"
msgstr "Tuto aplikaci musíte spouštět jako root"

#: ../tails_installer/creator.py:147
msgid "Extracting live image to the target device..."
msgstr "Rozbalování obrazu na cílové zařízení..."

#: ../tails_installer/creator.py:154
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "Zapsáno na zařízení %(speed)d MB/sec"

#: ../tails_installer/creator.py:184
msgid "Setting up OLPC boot file..."
msgstr "Nastavuji startovací soubor OLPC..."

#: ../tails_installer/creator.py:315
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "Vyskytl se problém s vykonáním následujícího příkazu: '%(command)s'\nPodrobnosti o této chybě byly zapsány do souboru '%(filename)s'."

#: ../tails_installer/creator.py:334
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "Ověřuji kontrolní součet SHA1 obrazu LiveCD..."

#: ../tails_installer/creator.py:338
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "Ověřuji kontrolní součet SHA256 obrazu LiveCD..."

#: ../tails_installer/creator.py:354
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "Chyba: SHA1 vašeho LiveCD je neplatný. Tento program můžete spustit s parametrem --noverify abyste tuto kontrolu přeskočili."

#: ../tails_installer/creator.py:360
msgid "Unknown ISO, skipping checksum verification"
msgstr "Neznámé ISO, přeskakuji ověření kontrolního součtu"

#: ../tails_installer/creator.py:371
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr "Není dostatek místa na zařízení.\n%dMB ISO + %dMB trvalé uložiště > %dMB volné místo"

#: ../tails_installer/creator.py:378
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "Vytvářím %sMB překrytí trvalého úložiště"

#: ../tails_installer/creator.py:439
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "Nemohu zkopírovat %(infile)s do %(outfile)s: %(message)s"

#: ../tails_installer/creator.py:453
msgid "Removing existing Live OS"
msgstr "Odebírám existující Live OS"

#: ../tails_installer/creator.py:462 ../tails_installer/creator.py:473
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "Nemohu změnit oprávnění %(file)s: %(message)s"

#: ../tails_installer/creator.py:466
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "Nemohu odebrat soubor z předchozího LiveOS: %(message)s"

#: ../tails_installer/creator.py:478
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "Nemohu odebrat složku z předchozího LiveOS: %(message)s"

#: ../tails_installer/creator.py:526
#, python-format
msgid "Cannot find device %s"
msgstr "Nemohu nalézt zařízení %s"

#: ../tails_installer/creator.py:695
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "Nemohu zapisovat na %(device)s, přeskakuji."

#: ../tails_installer/creator.py:719
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr "Některé oddíly, které jsou připojeny na cílovém zařízení %(device)s, budou před zahájením procesu instalace odpojeny."

#: ../tails_installer/creator.py:762 ../tails_installer/creator.py:974
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "Neznámý souborový systém. Vaše zařízení bude nutné zformátovat."

#: ../tails_installer/creator.py:765 ../tails_installer/creator.py:977
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Nepodporovaný systém souborů: %s"

#: ../tails_installer/creator.py:782
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr "Neznámá výjimka GLib při pokusu o připojení zařízení: %(message)s"

#: ../tails_installer/creator.py:786
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "Nemohu připojit zařízení: %(message)s"

#: ../tails_installer/creator.py:791
msgid "No mount points found"
msgstr "Nenalezeny žádné body připojení"

#: ../tails_installer/creator.py:802
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr "Spouští se unmount_device pro '%(device)s'"

#: ../tails_installer/creator.py:812
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr "Odpojuje se připojený souborový systém na '%(device)s'"

#: ../tails_installer/creator.py:816
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr "Odpojuje se '%(udi)s' z '%(device)s'"

#: ../tails_installer/creator.py:826
#, python-format
msgid "Mount %s exists after unmounting"
msgstr "Připojení %s existuje po odpojení"

#: ../tails_installer/creator.py:839
#, python-format
msgid "Partitioning device %(device)s"
msgstr "Vytváří se oddíly na zařízení %(device)s"

#: ../tails_installer/creator.py:959
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr "Nepodporované zařízení  '%(device)s', prosím nahlaste chybu"

#: ../tails_installer/creator.py:962
msgid "Trying to continue anyway."
msgstr "Snažím se pokračovat dál."

#: ../tails_installer/creator.py:971 ../tails_installer/creator.py:1364
msgid "Verifying filesystem..."
msgstr "Ověřuji systém souborů..."

#: ../tails_installer/creator.py:995
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "Není možné změnit jmenovku svazku: %(message)s"

#: ../tails_installer/creator.py:1000 ../tails_installer/creator.py:1397
msgid "Installing bootloader..."
msgstr "Instaluji bootloader..."

#: ../tails_installer/creator.py:1026
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr "Nelze najít '%s' COM32 modul"

#: ../tails_installer/creator.py:1034 ../tails_installer/creator.py:1415
#, python-format
msgid "Removing %(file)s"
msgstr "Odebírám %(file)s"

#: ../tails_installer/creator.py:1146
#, python-format
msgid "%s already bootable"
msgstr "%s je bootovatelné"

#: ../tails_installer/creator.py:1166
msgid "Unable to find partition"
msgstr "Nemohu najít oddíl"

#: ../tails_installer/creator.py:1189
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "Formátuji %(device)s jako FAT32"

#: ../tails_installer/creator.py:1249
msgid "Could not find syslinux' gptmbr.bin"
msgstr "Nelze najít syslinux gptmbr.bin"

#: ../tails_installer/creator.py:1262
#, python-format
msgid "Reading extracted MBR from %s"
msgstr "Čtení extrahované MBR z %s"

#: ../tails_installer/creator.py:1266
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr ""

#: ../tails_installer/creator.py:1279 ../tails_installer/creator.py:1280
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "Resetuji hlavní bootovací záznam %s"

#: ../tails_installer/creator.py:1285
msgid "Drive is a loopback, skipping MBR reset"
msgstr "Zařízení je smyčka, přeskakuji resetování hlavního bootovacího záznamu"

#: ../tails_installer/creator.py:1289 ../tails_installer/creator.py:1543
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "Kalkuluji SHA1 %s"

#: ../tails_installer/creator.py:1314
msgid "Synchronizing data on disk..."
msgstr "Synchronizuji data na disk..."

#: ../tails_installer/creator.py:1356
msgid "Error probing device"
msgstr "Chyba při snímání zařízení"

#: ../tails_installer/creator.py:1358
msgid "Unable to find any supported device"
msgstr "Nenalezeno žádné podporované zařízení."

#: ../tails_installer/creator.py:1368
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Ujistěte se, že váš USB disk je připojen a naformátován na systéme souborů FAT"

#: ../tails_installer/creator.py:1371
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "Nepodporovaný systém souborů: %s\nProsím proveďte zálohu a naformátujte USB disk systémem souborů FAT."

#: ../tails_installer/creator.py:1438
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr "Nemohu získat Win32_LogicalDisk; win32com výsledky"

#: ../tails_installer/creator.py:1491
msgid "Cannot find"
msgstr "Nemohu nalézt"

#: ../tails_installer/creator.py:1492
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr ""

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Neznámé vydání: %s"

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "Stahuji %s..."

#: ../tails_installer/gui.py:211
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "Chyba: Nelze nastavit jmenovku nebo získat UUID vašeho zařízení. Nemohu pokračovat."

#: ../tails_installer/gui.py:258
#, python-format
msgid "Installation complete! (%s)"
msgstr "Instalace kompletní! (%s)"

#: ../tails_installer/gui.py:263
msgid "Tails installation failed!"
msgstr "Instalace Tails selhala"

#: ../tails_installer/gui.py:363
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "Upozornění: Tento nástroj je nutné spustit jako Administrátor. Učiníte tak kliknutím pravým tlačítkem na ikonu a vyberte Vlastnosti. V záložce Kompatibilita vyberte možnost Spustit tento program jako správce."

#: ../tails_installer/gui.py:375
msgid "Tails Installer"
msgstr "Instalátor Tails"

#: ../tails_installer/gui.py:456
msgid "Upgrade"
msgstr ""

#: ../tails_installer/gui.py:462 ../data/tails-installer.ui.h:7
msgid "Install"
msgstr "Instalace"

#: ../tails_installer/gui.py:469
#, python-format
msgid "%(size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:481
msgid "No ISO image selected"
msgstr ""

#: ../tails_installer/gui.py:482
msgid "Please select a Tails ISO image."
msgstr ""

#: ../tails_installer/gui.py:521
msgid "No device suitable to install Tails could be found"
msgstr ""

#: ../tails_installer/gui.py:523
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr "Připojte prosím USB flash disk nebo SD kartu velikosti alespoň %0.1f GB."

#: ../tails_installer/gui.py:557
#, python-format
msgid ""
"The USB stick \"%(pretty_name)s\" is configured as non-removable by its "
"manufacturer and Tails will fail to start on it. Please try installing on a "
"different model."
msgstr ""

#: ../tails_installer/gui.py:567
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr "Zařízení \"%(pretty_name)s\" je moc malé pro instalaci Tails (je vyžadováno alespoň %(size)s GB)."

#: ../tails_installer/gui.py:589
msgid "An error happened while installing Tails"
msgstr ""

#: ../tails_installer/gui.py:601
msgid "Refreshing releases..."
msgstr "Obnovování verzí..."

#: ../tails_installer/gui.py:606
msgid "Releases updated!"
msgstr "Verze obnoveny!"

#: ../tails_installer/gui.py:648
msgid "Installation complete!"
msgstr "Instalace kompletní!"

#: ../tails_installer/gui.py:649
msgid "Installation was completed."
msgstr "Instalace byla dokončena."

#: ../tails_installer/gui.py:698
msgid "Unable to mount device"
msgstr "Nemohu připojit zařízení"

#: ../tails_installer/gui.py:705 ../tails_installer/gui.py:735
msgid "Confirm the target USB stick"
msgstr ""

#: ../tails_installer/gui.py:706
#, python-format
msgid ""
"%(size)s %(vendor)s %(model)s device (%(device)s)\n"
"\n"
"All data on this USB stick will be lost."
msgstr ""

#: ../tails_installer/gui.py:722
#, python-format
msgid "%(parent_size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:730
msgid ""
"\n"
"\n"
"The persistent storage on this USB stick will be preserved."
msgstr ""

#: ../tails_installer/gui.py:731
#, python-format
msgid "%(description)s%(persistence_message)s"
msgstr ""

#: ../tails_installer/gui.py:781
msgid "Download complete!"
msgstr "Stahování dokončeno!"

#: ../tails_installer/gui.py:785
msgid "Download failed: "
msgstr "Stahování selhalo: "

#: ../tails_installer/gui.py:786
msgid "You can try again to resume your download"
msgstr "Zkuste znovu zahájit stahování"

#: ../tails_installer/gui.py:794
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "Vybraný soubor je nečitelný. Opravte jeho oprávnění nebo vyberte jiný soubor."

#: ../tails_installer/gui.py:800
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "Nemohu použít vybraný soubor. Zkuste přesunout ISO do kořenového adresáře vašeho disku (např. C:\\)"

#: ../tails_installer/gui.py:806
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s zvolen"

#: ../tails_installer/source.py:28
msgid "Unable to find LiveOS on ISO"
msgstr "Nelze na ISO najít LiveOS"

#: ../tails_installer/source.py:34
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr ""

#: ../tails_installer/source.py:49
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr ""

#: ../tails_installer/source.py:63
#, python-format
msgid "'%s' does not exist"
msgstr "'%s' neexistuje"

#: ../tails_installer/source.py:65
#, python-format
msgid "'%s' is not a directory"
msgstr "'%s' není adresář"

#: ../tails_installer/source.py:75
#, python-format
msgid "Skipping '%(filename)s'"
msgstr "Přeskočit '%(filename)s'"

#: ../tails_installer/utils.py:44
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr "Nastal problém při spuštění `%s`.%s\n%s"

#: ../tails_installer/utils.py:124
msgid "Could not open device for writing."
msgstr "Nelze otevřít zařízení pro zápis."

#: ../data/tails-installer.ui.h:1
msgid "Installation Instructions"
msgstr ""

#: ../data/tails-installer.ui.h:2
msgid "Clone the current Tails"
msgstr ""

#: ../data/tails-installer.ui.h:3
msgid "Use a downloaded Tails ISO image"
msgstr ""

#: ../data/tails-installer.ui.h:4
msgid "Select a distribution to download:"
msgstr "Vyberte distribuci ke stažení:"

#: ../data/tails-installer.ui.h:5
msgid "Target USB stick:"
msgstr ""

#: ../data/tails-installer.ui.h:6
msgid "Reinstall (delete all data)"
msgstr ""
